### DISCUSSION TITLE
Using Homebrew on air-gapped network

### DISCUSSION THREAD

**Discussion by cddwyer** [Homebrew/discussions#4185] [Everyday usage]
### Output of `brew config`

```shell
HOMEBREW_VERSION: 3.6.21
ORIGIN: https://github.com/Homebrew/brew
HEAD: 932d2cf3b77c9439a57b6a43577fc8d3b6399a62
Last commit: 21 hours ago
Core tap origin: https://github.com/Homebrew/homebrew-core
Core tap HEAD: a9f91c4de40e2b91ac606bf651be034f9936ae88
Core tap last commit: 2 hours ago
Core tap branch: master
Core tap JSON: 08 Feb 13:23 UTC
HOMEBREW_PREFIX: /usr/local
HOMEBREW_CASK_OPTS: []
HOMEBREW_MAKE_JOBS: 4
Homebrew Ruby: 2.6.8 => /usr/local/Homebrew/Library/Homebrew/vendor/portable-ruby/2.6.8_1/bin/ruby
CPU: quad-core 64-bit broadwell
Clang: 13.0.0 build 1300
Git: 2.32.0 => /Library/Developer/CommandLineTools/usr/bin/git
Curl: 7.64.1 => /usr/bin/curl
macOS: 11.7.3-x86_64
CLT: 13.2.0.0.1.1638488800
Xcode: N/A
```

### Output of `brew doctor`

```shell
Your system is ready to brew.
```

### Description of issue

I work for an organisation looking to use homebrew in an air-gapped enviornment, so we would like to internally mirror all homebrew packages/have our own cellar. Is it possible to setup either on a macOS or Ubuntu server so macOS machines on our developer LAN can brew install pulling all packages and dependencies from our mirror instead of going out to the internet? I've read a lot of documentation and seem to be getting conflicting information from different sources. If someone could point me in the right direction I would be extremely grateful.

**Comment by SMillerDev** [comment]
The right direction for documentation is https://docs.brew.sh
